sandy_sort_lists.py

```diff
@@ -0,0 +1,94 @@
+import config
+import requests
```

Comment:
Great that you played around with `requests` and sending your own HTTP requests to their API!

From this, you must have noticed the "inconvenience" of having to hardcode the URLs and also pass in the parameters each time (with the credentials and what not). Oh, and what if the Trello API changes or maybe you need to add a new URL? You have to do it all again. Sound repetitive, right?

This is exactly what API "bindings" do for us. They're usually a light class that's language specific, and encapsulates all the details of interacting with the rest API. So e.g. you would use "Trello Python bindings", "Trello Ruby bindings", "Stripe Java bindings", etc.

Depending on the API the company may release an official one, or there may be open source variants out there. There usually is.

BOOM! https://trello.com/c/Thb6wfjZ/40-known-wrappers-projects-using-the-trello-api

Don't feel bad for not knowing this and having done it this way though. This was absolutely necessary to see what the "point" of a bindings library even is. Now you understand! (Took me a while to piece things together haha).

Does that make sense?